changelog/next/features/3684--contextualizer-backend.md

```diff
@@ -0,0 +1,8 @@
+The closed-source `contextualizer` plugin offers a backend functionality for
+finding matches between data sets.
+
+The open-source `lookup-table-context` plugin is a hashtable-based
```

Comment:
Should this one just be called `lookup-table`? There is no need for the suffix, I think.

Reply:
The plugin that the user has to specify when launching Tenzir is called `lookup-table-context`.

Reply:
IMO, `lookup-table` is too general a name for a plugin that does a very specific thing using lookup tables. 

If it was a builtin like `partitions` and `xsv`, we could easily put it into a folder `contexts` or something like that.

Reply:
See https://github.com/tenzir/tenzir/pull/3684#discussion_r1408307197 for the built-in discussion. I'm still in favor of it.

I have a strong preference on being consistent here. We separate between the name and the types of a plugin already, so there's little need to duplicate the type into the name. If we ever want to make the lookup-table plugin of another plugin type, then we'd need to rename the plugin, which is a breaking change.

Reply:
I'll make this a built-in.

Reply:
I made it a built-in in https://github.com/tenzir/tenzir/pull/3684/commits/ff65b1e9abefc6896f2e8d46eb3ad24c058bf6ad